picoquic/logwriter.c

```diff
@@ -822,6 +829,42 @@ void binlog_close_connection(picoquic_cnx_t * cnx)
     fflush(f);
 
     cnx->f_binlog = picoquic_file_close(cnx->f_binlog);
+
```

Comment:
Would it be possible to have a #define here such that you can exclude the QLOG stuff from picoquic for people who like the "pico" in picoquic? :) This also includes all the additional files you added to the picoquic core library for the QLOG conversion.

Reply:
I did not use a #define. Instead, I used a function pointer. I moved the files back to the log library, and added there an autoqlog file that defines `picoquic_set_qlog` and documents the function pointer. Applications that want to use that API will have to link the log library, and those that do without get a smaller code size.

Reply:
@steschu77 I think that fixes the code size issue that you mentioned. Will further reduce it once we get rid of the text log.

Reply:
Thanks, @huitema ! That is exactly what I was looking for!